src/workload.rs

```diff
@@ -487,6 +497,18 @@ impl WorkloadInformation {
     }
 }
 
+impl Default for Box<dyn CertificateProvider> {
+    fn default() -> Self {
+        Box::new(MockCaClient::new(Duration::from_secs(5)))
+    }
+}
+
+impl std::fmt::Debug for Box<dyn CertificateProvider> {
```

Comment:
This seems suspicious? why are we emitting just `""`?

Reply:
removed, not sure why i put that there